src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css

```diff
@@ -643,3 +610,78 @@ aside.topic,
 div.versionchanged {
   border-color: var(--ast-admonition-neutral-border);
 }
+
+/**
+* Dropdown _header.scss
+*/
+.bd-header ul.navbar-nav > li.nav-item.dropdown > .dropdown-toggle {
+  color: var(--ast-navbar-color);
+  font-weight: 600;
+  font-size: var(--bs-nav-link-font-size);
+  border-radius: 4px;
+  border-color: var(--ast-dropdown-header-border);
+}
+
+.bd-header ul.navbar-nav > li.nav-item.dropdown > .dropdown-toggle:hover {
+  color: var(--ast-navbar-hover-color);
+  border-bottom: none;
+  box-shadow: var(--ast-box-shadow-hover);
+}
+
+.bd-header ul.navbar-nav .dropdown .dropdown-menu .dropdown-item {
+  color: var(--ast-navbar-color);
+  font-size: var(--bs-nav-link-font-size);
+  font-weight: 600;
+  border-radius: 4px;
+  border-color: var(--ast-dropdown-header-border);
+}
+
+/**
+* Dropdown _header.scss
+*/
+
+.version-switcher__menu a.list-group-item {
+  background-color: var(--ast-color-enable-card-background);
+  width: 200px;
+  border: none;
+  filter: drop-shadow(var(--ast-box-shadow-active));
+  color: var(--ast-navbar-hover-color);
+  font-size: var(--ast-nav-link-font-size);
+  height: 40px;
+  padding: 0px 16px;
+  text-decoration: none;
+  border-radius: 4px;
+}
+
+button.btn.version-switcher__button {
+  background-color: transparent;
+  border-color: var(--ast-navbar-active-border-color);
+  color: var(--ast-navbar-color);
+  font-size: var(--ast-nav-link-font-size);
+}
+
+button.btn.version-switcher__button:hover {
+  border-color: var(--ast-navbar-active-border-color);
+  color: var(--ast-navbar-hover-color);
+}
+
+.version-switcher__menu a.list-group-item:hover {
+  background-color: var(--ast-color-hover-card-background);
+  border: none;
```

Comment:
I feel like both dropdowns (version and "more") should be have a border. Right now, the "more" dropdown has one but the version one does not.

What is your feedback on this, @mia-guo-ux ?

Reply:
I removed it according to the suggestion in [this GitHub issue](https://github.com/ansys/ansys-sphinx-theme/issues/403#issuecomment-2179523375), but I'm happy to revert it if needed.

Reply:
Thanks for the explanation, @Revathyvenugopal162. Just wanted to double check with Mia about this.

Reply:
Thanks for bringing it up :) In our design system, we do not have a border for dropdown menus. Instead, we apply drop shadows to indicate the hierarchy and to distinguish it from the background. Please let me know if you have any questions.

Reply:
Thanks for the clarification, Mia.